src/kuzzle.js

```diff
@@ -1032,8 +1032,6 @@ Kuzzle.prototype.listIndexes = function (options, cb) {
 Kuzzle.prototype.disconnect = function () {
   var collection;
 
-  this.logout();
```

Comment:
After a quick discussion with @scottinet and @jenow, we decided that if the user want to logout, he can directly call `kuzzle.logout()` explicitely.